src/node_manager/model/cli.rs

```diff
@@ -26,8 +26,8 @@ pub struct Status {
 
 impl std::fmt::Display for Status {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
-        writeln!(f, "Connected Peers Count:   {}", self.peers_count);
-        writeln!(f, "Artifacts Count:         {}", self.artifact_count);
+        if let Err(_e) = writeln!(f, "Connected Peers Count:   {}", self.peers_count) {}
+        if let Err(_e) = writeln!(f, "Artifacts Count:         {}", self.artifact_count) {}
```

Comment:
The writeln!s are not conditional in the original version. This looks like a change to the logic, not just a clippy fix.

Reply:
The warning was this:
```
warning: unused `std::result::Result` that must be used
```

We were implicitly ignoring the Err variant of the result, which triggered the warning. So I decided to explicitly handle and ignore the error.

Maybe even better would be to compose the full string and write it using 1 write! macro, so the error can be properly returned as well? (which it already does now in the last of the 3 write's)

Reply:
The error of the last `writeln` is indeed propagated back to the caller. The simplest way to have the same behavior on the first two statements is to use the `?` operator.
```suggestion
        writeln!(f, "Artifacts Count:         {}", self.artifact_count)?;
```